code/src/scenes/movie/Movie.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useParams } from 'react-router-dom';
+import { Grid, Typography } from '@material-ui/core';
+
+import { MOVIE_URL, BASE_IMAGE_POSTER_URL } from './../../constants';
+import { SimilarMovies } from './../../components/SimilarMovies';
+
+const useStyles = makeStyles((theme) => ({
+    poster: {
+        maxWidth: '100%'
+    },
+}));
+
+export const Movie = () => {
+    let { id } = useParams();
+    const classes = useStyles();
+
+    const [movie, setMovie] = useState({})
+    const [loading, setLoading] = useState(true)
```

Comment:
It would be more appropriate to initialise `loading` with value `false` - change it to `true` in the `useEffect`, before `fetch`!